spec/lib/open_food_network/feature_toggle_spec.rb

```diff
@@ -2,17 +2,17 @@
 
 require 'spec_helper'
 
-module OpenFoodNetwork
-  describe FeatureToggle do
-    context 'when users are not specified' do
-      it "returns false when feature is undefined" do
-        expect(FeatureToggle.enabled?(:foo)).to be false
-      end
+describe OpenFoodNetwork::FeatureToggle do
+  subject(:feature_toggle) { OpenFoodNetwork::FeatureToggle }
 
-      it "uses Flipper configuration" do
-        Flipper.enable(:foo)
-        expect(FeatureToggle.enabled?(:foo)).to be true
-      end
+  context 'when users are not specified' do
+    it "returns false when feature is undefined" do
+      expect(feature_toggle.enabled?(:foo)).to be false
```

Comment:
I think `described_class` would have also worked :)

Reply:
I know, but it didn't read that well.